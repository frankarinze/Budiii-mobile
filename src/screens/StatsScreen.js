import React from 'react';
import { ScrollView, Text, View } from 'react-native';
import { gStyle } from '../constants';

const StatsScreen = () => (
  <View style={gStyle.container}>
    <ScrollView contentContainerStyle={gStyle.contentContainer}>
      <Text style={gStyle.text}>Stats Content Area</Text>
    </ScrollView>
  </View>
);

StatsScreen.navigationOptions = {
  headerTitleStyle: {
    flex: 1,
    textAlign: 'center'
  },
  title: 'Stats'
};

export default StatsScreen;
