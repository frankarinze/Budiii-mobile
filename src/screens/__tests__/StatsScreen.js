import React from 'react';
import renderer from 'react-test-renderer';

import StatsScreen from '../StatsScreen';

const navigation = {
  goBack: jest.fn(),
  navigate: jest.fn()
};

const tree = renderer.create(<StatsScreen navigation={navigation} />).toJSON();

describe('<StatsScreen />', () => {
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
