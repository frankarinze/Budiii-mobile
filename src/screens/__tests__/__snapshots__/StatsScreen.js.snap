// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatsScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
/>
`;
